src/shared/conf-parser.c

```diff
@@ -642,7 +639,7 @@ int config_parse_config_file(
 
         const char *sysconf_file = strjoina(SYSCONF_DIR, "/", conf_file);
 
-        return config_parse_many_files(STRV_MAKE_CONST(sysconf_file), dropins,
+        return config_parse_many_files(NULL, STRV_MAKE_CONST(sysconf_file), dropins,
```

Comment:
maybe add `/* root= */`

Reply:
This chunk is removed later on, so I don't think this matters. But I added it to some other calls that appear in the final version, so you should be satisfied ;)